vta/src/runtime.cc

```diff
@@ -290,20 +295,20 @@ class UopKernel {
 /*!
  * \brief Base class of all queues to send and recv serial data.
  */
+template <class T>
 class BaseQueue {
  public:
   ~BaseQueue() {
-    if (dram_buffer_ != nullptr) {
-      VTAMemFree(dram_buffer_);
-    }
+    VTAMemFree(fpga_buff_);
```

Comment:
Do we need to check the handle is not null?

Reply:
agreed, this should have a check